workflows-scripts/data-export/README.md

```diff
@@ -0,0 +1,43 @@
+## Workflow steps:
+
+##### Steps to generate data:
+1. Open Inventory App -> Instance tab -> select query and type for example: "source= marc" to get search records
```

Comment:
Is this part of the workflow as well? Looks like it's a step in the worfklow

Reply:
Yes, I have updated it.